app/src/main/res/layout/profile_reset_pin_activity.xml

```diff
@@ -99,6 +101,7 @@
             android:layout_marginStart="28dp"
             android:layout_marginTop="28dp"
             android:layout_marginEnd="28dp"
+            android:textDirection="locale"
```

Comment:
ditto here

Reply:
@ShridharGoel Please reply to these comments.

Reply:
This has been updated.